docs/ruleset_functions/is_zone_a_vestibule.md

```diff
@@ -0,0 +1,51 @@
+# is_zone_a_vestibule
+**Schema Version:** 0.0.22  
+
+**Description:** following the guidelines in ASHRAE that a vestibule is defined as a sapce with at least one exterior door and with a surface area of no more than the greater of 50ft2 or 2% of the total area of the floor.  There is no 100% check for a vestibule, so a space that meets these requirements and also has only 6 surfaces (floor, ceiling and 4 walls) will return "LIKELY" - one that meets the official requirements but has more surfaces will return "POSSIBLY"

```

Comment:
My understanding is that the 2% is 2% of the building floor area:

5.4.3.3.2 Vestibule Size
Vestibules shall be designed so that in passing through the vestibule it is not necessary for
the interior and exterior doors to open at the same time. Interior and exterior doors shall
have a minimum distance between them of not less than 7 ft when in the closed position.
The floor area of each vestibule shall not exceed the greater of 50 ft2 or 2% of the gross conditioned
floor area for that level of the building.

Reply:
Regarding six surfaces - this would mean one floor, one ceiling and four walls.  Vestibules often have only six surfaces, but corridors almost never do, and they are adjacent to a lot more zones, so this was an attempt to eliminate corridors, but it is not an exact science.

Reply:
I added a check for lighting.  This happens at the beginning of the function and only zones that contain all spaces that have compliant lighting_space_types continue with the rest of the vestibule checks